test/integration/puppetlabs/services/jruby/puppet_environments_int_test.clj

```diff
@@ -57,20 +57,6 @@
     {:headers     {"Accept" "pson"}
      :as          :text}))
 
-(defn service-context
```

Comment:
I had a need to use these for some of the lockable pool tests, so I moved these down into the common `test/unit/puppetlabs/services/jruby/jruby_testutils`.